### ISSUE TITLE
mss pyui crashes when quitting

### ISSUE THREAD

**Issue by ReimarBauer** [Open-MSS/MSS#505]
**[Original report](https://bitbucket.org/wxmetvis/mss/issue/505) by Joern Ungermann (Bitbucket: [joernu76](https://bitbucket.org/joernu76), GitHub: [joernu76](https://github.com/joernu76)).**

----------------------------------------

When MSS is quit without opening an msolab window it crashes with the error message:

```
Fatal error in MSS 1.9.0 on Linux-4.19.0-6-amd64-x86_64-with-debian-10.1
Python 3.7.4 (default, Aug 13 2019, 20:35:49) 
[GCC 7.3.0]

Please report bugs in MSS to https://bitbucket.org/wxmetvis/mss

Information about the fatal error:

Traceback (most recent call last):
  File "mss_pyui.py", line 424, in closeEvent
    if self.mscolab_window is not None:
AttributeError: 'MSSMainWindow' object has no attribute 'mscolab_window'
```

I think it should also crash in case that the mscolab window was first opened and then closed - it didn’t when testing for this just now, but I do not see how the reference to the window is set to none by anybody.

‌

I deem this critical as every “normal” user will experience this crash undermining the trust in the software.

**Comment by ReimarBauer** [comment]
**Original comment by Shivashis Padhi (Bitbucket: [plant99](https://bitbucket.org/plant99), GitHub: [plant99](https://github.com/plant99)).**

----------------------------------------

This bug is introduced in the recent commit, I'll push a fix soon.

**Comment by ReimarBauer** [comment]
**Original comment by Shivashis Padhi (Bitbucket: [plant99](https://bitbucket.org/plant99), GitHub: [plant99](https://github.com/plant99)).**

----------------------------------------

Made a PR

**Comment by ReimarBauer** [comment]
**Original comment by Joern Ungermann (Bitbucket: [joernu76](https://bitbucket.org/joernu76), GitHub: [joernu76](https://github.com/joernu76)).**

----------------------------------------

Sorry, for not claiming the bug immediately. Made my PR in a hurry before leaving for the airport.

PRs seem to be sorted our.

**Comment by ReimarBauer** [comment]
**Original comment by Shivashis Padhi (Bitbucket: [plant99](https://bitbucket.org/plant99), GitHub: [plant99](https://github.com/plant99)).**

----------------------------------------

It is no problem, didn't take much time anyway.

‌

And yes, it’s sorted. Only merging remains, @{557058:3ffd4910-450c-422e-9ab9-726d7f2c7449}

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

initialize reference to mscolab window with none

Fix issue #505
